src/components/va-back-to-top/test/va-back-to-top.e2e.ts

```diff
@@ -87,82 +40,75 @@ describe('va-back-to-top', () => {
   });
 
   it('reveals when the viewport is below the reveal pixel', async () => {
-    const page = await newE2EPage();
-    await page.setContent(`
-    <main>
-    ${pageContent}
-    <va-back-to-top></va-back-to-top>
-    </main>
-    `);
+    const page = await pageSetup();
     const button = await page.find('va-back-to-top >>> button');
+    const notQuitePastRevealPoint = 199;
 
     // Button shouldn't be revealed initially
     expect(button).not.toHaveClass('reveal');
 
-    await page.mouse.wheel({ deltaY: 100 });
+    // Scroll until the `<span class="reveal-point">` is just at
+    // the top of the viewport
+    await page.mouse.wheel({ deltaY: notQuitePastRevealPoint });
     await page.waitForChanges();
 
     // Button _still_ shouldn't be revealed after a bit of scrolling
     expect(button).not.toHaveClass('reveal');
 
-    await page.mouse.wheel({ deltaY: 1000 });
+    // Scroll a few pixels more to get past the `--reveal-breakpoint`
+    await page.mouse.wheel({ deltaY: 10 });
     await page.waitForChanges();
 
-    // We've scrolled a lot - button should be revealed
+    // Button should be revealed with `.reveal-point` above the viewport
     expect(button).toHaveClass('reveal');
   });
 
   it('docks when the "dock" is scrolled into view', async () => {
-    const page = await newE2EPage();
-    await page.setContent(`
-    <main>
-    ${pageContent}
-    <va-back-to-top></va-back-to-top>
-    <footer style="height: 100px">The footer</footer>
-    </main>
-    `);
+    const page = await pageSetup();
     const wrapper = await page.find('va-back-to-top >>> div');
+    const pastPlaceholder = 1000;
```

Comment:
Same here: If we had a `revealPoint` variable, we could set this to `revealPoint + 1000` or whatever.